src/Config/DataApiBuilderException.cs

```diff
@@ -102,6 +102,14 @@ public enum SubStatusCodes
         /// DataSource not found for multiple db scenario.
         /// </summary>
         DataSourceNotFound,
+        /// <summary>
+        /// Item identified by the PK is not found in the database.
+        /// </summary>
+        ItemNotFound,
+        /// <summary>
+        /// Invalid PK field(s) specified in the request.
+        /// </summary>
+        InvalidPrimaryKeyField
```

Comment:
nit: we abstract away database terminology from the user. I suggest using `InvalidIdentifierField`